src/Components/Products/Products.css

```diff
@@ -0,0 +1,233 @@
+.main-container{
+    display: flex;
+    width: 100%;
+    flex-direction: row;
+    align-items: stretch;
+    margin: 0;
+}
+
+/* aside */
+.side-content{
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    width: 16%;
+    background-color: var(--lg-color);
+    margin: 1rem;
+}
+.main-content{
+    width: 80%;
+}
+.side-title{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+.filter-btn{
+    background-color: transparent;
+    border-radius: 0;
+    font-weight: bolder;
+    color: var(--font--secondary-color);
+    padding: 0;
+}
+.filter-btn:hover{
+    background-color: transparent;
+}
+.clear-btn{
+    background-color: transparent;
+    color: var(--secondary-hover);
+    border-radius: 0;
+    padding: 0;
+}
+.clear-btn:hover{
+    background-color: transparent;
+}
+.filter-btn:hover,.clear-btn:hover{
+    border-bottom: 2px solid var(--primary-color);;
+}
+.filter-category{
+    padding: 0.8rem;
+    font-weight: bold;
+    font-size: 0.9rem;
+}
+.options{
+    padding: 0.1rem 0.4rem;
+    font-size: 0.9rem;
+}
+.options label{
+    display: block;
+}
+.options label input{
+    margin: 0 0.3rem;
+}
+/* main */
+
+/* product card */
+.grid-3-col{
+    margin: 2rem 1rem;
+    display: grid;
+    /* grid-template-columns: repeat(3,1fr); */
+    grid-template-columns:repeat(auto-fill, minmax(15rem, 1fr));
+    grid-gap:1.5rem;
+    max-width: 80rem;
+
+}
+
+.card-vertical{
+    display: flex;
+    flex-direction: column;
+    max-width: 14rem;
+    width: 100%;
+    background-color: var(--lg-color);
+    cursor: pointer;
+    box-shadow: 2px 5px 5px rgba(0,0,0,0.25);
+    position: relative;
+}
+.card-vertical:hover{
+    box-shadow: 6px 6px 8px #888888;
+}
+.res-img{
+    width: 100%; 
+    /* max-width: 14rem; */
+    height: 15rem;
+    /* max-width: 100%;
+    height: auto; */
+    object-fit: cover;
+  }
+.text-container{
+    padding: 0 0.5rem;
+}
+.title-wishlist{
+    display: flex;
+    justify-content: space-between;  
+}
+.text-container-title{
+    font-weight: bold;
+    color: var(--secondary-hover)
```

Comment:
Please do add semi-colon here as it's not optional in CSS

Reply:
done